### PR TITLE
feat: Creates component to merge contacts

### PR DESCRIPTION
**Pull request by nithindavid**
image

### CODE REVIEW

**Comment by sojan-official** [comment]
ref : https://github.com/chatwoot/chatwoot/issues/258

File: app/javascript/dashboard/modules/contact/components/MergeContact.vue [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+          </label>
+          <multiselect
+            v-model="childContact"
+            :options="searchResults"
+            label="name"
+            track-by="id"
+            :internal-search="false"
+            :clear-on-select="false"
+            :show-labels="false"
+            placeholder="Choose a contact"
```
Comment (pranavrajs) [inline app/javascript/dashboard/modules/contact/components/MergeContact.vue:48]: Please use i18n

File: app/javascript/dashboard/modules/contact/components/MergeContact.vue [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+            <template slot="singleLabel" slot-scope="props">
+              <thumbnail
+                :src="props.option.thumbnail"
+                size="24px"
+                :username="props.option.name"
+              />
+              <span class="option__title">
+                {{ props.option.name }}
+              </span>
+            </template>
```
Comment (pranavrajs) [inline app/javascript/dashboard/modules/contact/components/MergeContact.vue:64]: This is repeated above, can we create a component.

File: app/javascript/dashboard/modules/contact/components/MergeContact.vue [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+  mixins: [alertMixin],
+  props: {
+    primaryContact: {
+      type: Object,
+      required: true,
+    },
+    onContactSearch: {
+      type: Function,
+      default: () => {},
+    },
```
Comment (pranavrajs) [inline app/javascript/dashboard/modules/contact/components/MergeContact.vue:108]: Can we use $emit instead of passing the function as a prop?

Comment (pranavrajs) [reply app/javascript/dashboard/modules/contact/components/MergeContact.vue:108]: Also, I think it is not used.

File: app/javascript/dashboard/modules/contact/components/ContactDropdownItem.vue [hunk 4]
```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class="option-item--user">
+    <thumbnail :src="thumbnail" size="24px" :username="name" />
```
Comment (muhsin-k) [inline app/javascript/dashboard/modules/contact/components/ContactDropdownItem.vue:3]: The `<thumbnail>` component is not registered correctly.

Screenshot 2021-06-29 at 11 34 19 AM

**Pull request merged**